### PR TITLE
fix: adding cloudtype to cloudalertrecord gaia object

### PR DESCRIPTION
**Pull request by rramalinga**
## Description

Adding cloudtype to the `cloudalertrecord` object so that we can set the cloudtype in alert payload.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1591",
    "commit-sha": "8b03b52907057623570f76e9a3beda79844b548d",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "779",
    "commit-sha": "cf41251a10734b8edd4a097d85ab70ce8504bcd9",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1591",
    "commit-sha": "aad63fa9a91d1a90ab93117ee8df4c58bf93a8e3",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "779",
    "commit-sha": "cf41251a10734b8edd4a097d85ab70ce8504bcd9",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1591",
    "commit-sha": "aad63fa9a91d1a90ab93117ee8df4c58bf93a8e3",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "779",
    "commit-sha": "cf41251a10734b8edd4a097d85ab70ce8504bcd9",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1591",
    "commit-sha": "3421f197c3a95e527f93bd8267abddb076f5ef33",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "779",
    "commit-sha": "cf41251a10734b8edd4a097d85ab70ce8504bcd9",
    "pipeline": "master"
  }
]
```

File: specs/cloudalertrecord.spec [hunk 1]
```diff
@@ -47,6 +47,18 @@ attributes:
...
+  - name: cloudType
+    description: Cloud type of the entity.
+    type: string
+    exposed: true
+    stored: true
+    default_value: AWS
+    example_value: AWS
+    getter: true
+    setter: true
+    orderable: true
```
Comment (malkhamis-panw) [inline specs/cloudalertrecord.spec:59]: I don't have the full dev context, but I feel these three fields may not be necessary (getter, setter, orderable). Someone with better context might be able to confirm as I only mean to bring eyes here.

Cheers

Comment (sawyerawesome) [reply specs/cloudalertrecord.spec:59]: @rramalinga should I delete these fields?

**Review by malkhamis-panw — DISMISSED** [review]

File: specs/cloudalertrecord.spec [hunk 2]
```diff
@@ -47,6 +47,15 @@ attributes:
     subtype: string
     stored: true
 
+  - name: cloudType
+    description: Cloud type of the entity.
+    type: string
```
Comment (t00f) [inline specs/cloudalertrecord.spec:52]: Shouldn't it be an enum? With two options `AWS` and `Azure` ?

**Review by t00f — COMMENTED** [review]
Looks good. Just one question

**Review by t00f — APPROVED** [review]

**Pull request merged**